code/src/components/product-card/ProductCard.scss

```diff
@@ -38,4 +49,27 @@
     @include transition((all, "medium"));
     backdrop-filter: blur(spacing(2));
   }
+
+  &__footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: auto;
+    padding-top: spacing(2);
+    border-top: 1px solid var(--spa-products-container-border-color, $light);
+  }
+
+  &__button {
+    border: 1px solid var(--spa-best-sellers-price-color, $black);
+    background-color: transparent;
+    color: var(--spa-best-sellers-price-color, $black);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: spacing(1);
```

Comment:
I think you should use the existing variant of our `AppButton` and remove these styles. Maybe only a gap for the icon, but I'm not sure about it

Reply:
> I think you should use the existing variant of our `AppButton` and remove these styles. Maybe only a gap for the icon, but I'm not sure about it

I already used `AppButton` and also remove some useless styles for that